src/sdk/namespace_cache.js

```diff
@@ -48,7 +58,7 @@ class NamespaceCache {
 
     async list_objects(params, object_sdk) {
         // TODO listing from cache only for deevelopment
-        return this.namespace_nb.list_objects(params, object_sdk);
+        return this.namespace_hub.list_objects(params, object_sdk);
```

Comment:
Can we support listing from cache option through query parameter get_from_cache? If get_from_cache is set, it lists objects from cache.